Update interfaces/native-file-system.idl

autofoolip opened the pull request:
This PR was automatically created by a bot.

Before merging, please check that any tests that depend on the updated IDL files still work.

If additional changes are needed, please manually create another PR based on this one.

See the [README](https://github.com/web-platform-tests/wpt/blob/master/interfaces/README.md) for how the IDL files in this directory are used.

<hr>

Source: https://github.com/tidoust/reffy-reports/blob/a1daebd/ed/idl/native-file-system.idl
Build: https://travis-ci.org/tidoust/reffy-reports/builds/719056521

foolip reviewed (changes requested):
Oops, will require some fixes. A new failure is introduced:
> FAIL message: promise_test: Unhandled rejection with value: object "FileSystemPermissionDescriptor inherits PermissionDescriptor, but PermissionDescriptor is undefined."

foolip reviewed (changes requested):
Oops, actually no good, we have missing dependencies now.

autofoolip committed:
Update interfaces/native-file-system.idl

Source: https://github.com/tidoust/reffy-reports/blob/a1daebd/ed/idl/native-file-system.idl
Build: https://travis-ci.org/tidoust/reffy-reports/builds/719056521
